metriken-exposition/Cargo.toml

```diff
@@ -14,8 +14,8 @@ repository = "https://github.com/iopsystems/metriken"
 [dependencies]
 arrow = { version = "50.0.0", optional = true }
 chrono = "0.4.34"
-histogram = "0.9.1"
-metriken = { version = "0.5.1", path = "../metriken" }
+histogram = "0.10.0"
+metriken = { version = "0.6.0", path = "../metriken" }
```

Comment:
Out of curiosity, does the version actually apply when the path is provided? It always just seems to pick up whatever is at the location for me...

Reply:
For me it was complaining that it couldn't find a crate with a matching version. But we definitely need to update so we can publish, as published crates must have all their dependencies published.

Reply:
Ah, makes sense